### DISCUSSION TITLE
Using ESP32-ROVER-E instead of ESP32-WROOM

### DISCUSSION THREAD

**Discussion by ineptcamel** [darthcloud/BlueRetro#341] [Q&A] [answered]
My local surplus shop has some ESP32 DevKitC boards but they have the ROVER-E designation instead of WROOM. 

According to Espressif, the functional difference between these two is that ROVER has 8MB flash but WROOM has 4MB flash. Seems like it shouldn't be an issue right? 

Further, from Espressif,

````

The pins GPIO16 and GPIO17 are available for use only on the boards with the modules ESP32-WROOM and ESP32-SOLO-1. The boards with ESP32-WROVER modules have the pins reserved for internal use

````

IO17 appears to be used for the LED. IO16 is connected to the Dsub on the ESP32 side but most console cables don't seem to use it (or N64 and GCN anyway, which is all I would use).

So if I sacrifice the LED could I make one work with ROVER?

**Comment by darthcloud** [comment, marked as answer]
You could try but it might make PSRAM do weird thing. Just get a wroom?

**Reply by ineptcamel** [reply]
Thanks for the response! When I have more time I may screw around with it and try to make it work because they were dirt cheap.

For right now I'll stick to the two WROOM's on the way.
